components/strava/actions/get-stats/get-stats.mjs

```diff
@@ -0,0 +1,17 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Get Stats",
+  description: "Returns the activity stats of an athlete. Only includes data from activities set to Everyone visibilty. See `https://developers.strava.com/docs/reference/`",
+  key: "strava-get-stats",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    strava,
+  },
+  async run({ $ }) {
+    const resp = await this.strava.getStats($);
+    $.export("$summary", "The activity stats have been retrieved");
+    return resp;
```

Comment:
Once you make the refactor in `strava.app.js` you can apply these changes
```suggestion
    const { id: athleteId } = await this.strava.getAuthenticatedAthlete({
      $,
    });
    const resp = await this.strava.getAthleteStats({
      $,
      athleteId
    });
    $.export("$summary", "The activity stats have been retrieved");
    return resp;
```

Reply:
done with small changes